schema_salad/python_codegen.py

```diff
@@ -112,6 +118,19 @@ def prologue(self) -> None:
         stream.close()
         self.out.write("\n\n")
 
+        if self.parser_info:
+            info = self.parser_info
+        else:
+            info = ""
+
+        self.out.write(
+            f"""def parser_info() -> str:
+    return "{info}"
+
+
+"""
+        )
```

Comment:
```suggestion
        self.out.write(
            f"""def parser_info() -> str:
    return "{self.parser_info}"


"""
        )
```

Reply:
Unfortunately it accidentally returns `"None"` if `self.parser_info` is not provided :-(
I guess users expect an empty string rather than `"None"` in this case.

Reply:
`return "{self.parser_info or ""}"`

Reply:
Done!
I also added a test for it.